norduser/service/fork.go

```diff
@@ -170,6 +171,16 @@ func (c *ChildProcessNorduser) Stop(uid uint32) error {
 		return fmt.Errorf("sending SIGTERM to norduser process: %w", err)
 	}
 
+	if wait {
+		proc, err := os.FindProcess(pid)
+		if err == nil {
+			_, err := proc.Wait()
+			if err == nil {
```

Comment:
This check is not needed, you can ignore the error and function will return nil in line 184.